EnglishFilter/sections/foreign.txt

```diff
@@ -33,6 +33,9 @@ video.gjirafa.com#%#AG_defineProperty('originVideo.requestA.Id', { value: "0" })
 !------------------------------ Arabian ------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/78869
+asiatv.*#$#.ad-banner { display: block !important; }
+asiatv.*#@#.ad-banner
```

Comment:
Why do you use `*` here?
`asiatv.online#@#.ad-banner` is enough.

Reply:
The domain changes quite frequently.

Reply:
Changed to scriptlet. I don't like to unblock elements, unless necessary, for platforms which supports Scriptlets and JS.

Reply:
> Changed to scriptlet. I don't like to unblock elements, unless necessary, for platforms which supports Scriptlets and JS.

Save the scriptlet for the future, unblocking element is enough here.

Reply:
Why did you remove it? I thought that's the way issues are supposed to be fixed. Check other anti-adblock fixes, it has a scriptlet + a whitelist rule for platforms without scriptlets support.

Reply:
I've already explained it. [Unblocking element is enough here](https://github.com/AdguardTeam/AdguardFilters/pull/78899#discussion_r604619883). No need to complicate things, use an easier way to solve issues.
Using the scriptlet will increase the load on the extension, you need to use it when there are no other options.